extensions/wikia/UserLogin/js/mixins/UserSignup.mixin.js

```diff
@@ -0,0 +1,23 @@
+(function () {
```

Comment:
We haven't done a lot of JS mixins outside of VE, so this is slightly new for the MW app. We may want to set up some best practices around it at some point.